File: ifelseifelse-go/ifelseifelse.go
// ifelseifelse.go
package main

import "fmt"

func main() {
    x := 5
    if x == 10 {
        fmt.Println("x is 10")
    } else if x < 10 {
        fmt.Println("x is less than 10")
    } else {
        fmt.Println("x is greater than 10")
    }
}

File: ifelseifelse.zig
// ifelseifelse.zig
const std = @import("std");

pub fn main() void {
    const x: i32 = 5;
    if (x == 10) {
        std.debug.print("x is 10\n", .{});
    } else if (x < 10) {
        std.debug.print("x is less than 10\n", .{});
    } else {
        std.debug.print("x is greater than 10\n", .{});
    }
}
